docs/ml-toolkit/index.md

```diff
@@ -0,0 +1,6 @@
+# DeepESDL ML Toolkit Documentation
```

Comment:
Can we maybe skip Index entirely and start with Introduction? And add the Overview into the Introduction? This would need to be reflected in mkdocs.yml as well.